src/helpers.rs

```diff
@@ -119,9 +119,9 @@ pub fn get_jwt(token: String) -> Result<TokenData<Claims>, String> {
 }
 
 pub fn get_age(year: i32, month: u32, day: u32) -> f64 {
-    return (((SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis()
+    (((SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis()

```

Comment:
Unwrap ?

Reply:
I assume you mean I have to handle the unwrap?